src/main/java/com/msalvatore/graphqlplayground/entity/Vehicle.java

```diff
@@ -0,0 +1,76 @@
+package com.msalvatore.graphqlplayground.entity;
+
+import javax.persistence.*;
```

Comment:
Oops. I changed my IntelliJ settings after this. Checkstyle should have caught this but it's not hooked up yet :-)